shared/more/style-sheet.render.desktop.js

```diff
@@ -89,6 +92,81 @@ export default class Render extends Component {
   }
 }
 
+class DZ2Font extends Component {
+  render () {
+    const Space = () => <div style={{height: 20}}/>
+    return (
+      <div style={globalStyles.flexBoxColumn}>
+        <div style={globalStyles.flexBoxRow}>
+
+          {[false, true].map(darkMode => (
+            <div style={
+              {...globalStyles.flexBoxColumn,
+                flex: 1,
+                padding: 40,
+                backgroundColor: (darkMode ? globalColorsDZ2.darkBlue : globalColorsDZ2.white)}}>
+              <Text darkMode={darkMode} type='Header-Jumbo'>Header Jumbo</Text>
+              <Text darkMode={darkMode} type='Header-Jumbo'>Header Jumbo</Text>
+              <Space/>
+              <Text darkMode={darkMode} type='Header-Big'>Header Big Header Big</Text>
+              <Text darkMode={darkMode} type='Header-Big'>Header Big Header Big</Text>
+              <Space/>
+              <Text darkMode={darkMode} type='Header'>Header Header Header</Text>
+              <Text darkMode={darkMode} type='Header'>Header Header Header</Text>
+              <Space/>
+              <Text darkMode={darkMode} type='Body'>Body Body</Text>
+              <Text darkMode={darkMode} type='Body'>Body Body</Text>
+              <Space/>
+              <Text darkMode={darkMode} type='Body-Semibold'>Body Semibold Body Semibold</Text>
+              <Text darkMode={darkMode} type='Body-Semibold'>Body Semibold Body Semibold</Text>
+              <Space/>
+              <Text darkMode={darkMode} type='Body-Small'>Body small Body Small</Text>
+              <Text darkMode={darkMode} type='Body-Small'>Body small Body Small</Text>
+              <Space/>
+
+              <div style={{alignSelf: (darkMode ? 'flex-start' : 'flex-end')}}>
+                {(darkMode ? [1.0, 0.75, 0.4] : [0.75, 0.4, 0.1]).map(opacity => (
```

Comment:
The opacity needs to be captured as a prop to the Text and not something we specify as consumers directly. AKA Body small is always 0.4 / 0.75. its not something that's configurable so we should never see <Text style={opacity:...>